test/templates/product-tile/index.njk

```diff
@@ -72,6 +72,64 @@
       {% endset %}
         {{ format(example) }}
 
+<br><br>
 
+<h2> Example with H2 </h2>
```

Comment:
This page looks like it's missing the example with H5 (or it's not rendering properly).